photon-client/src/test/scala/com/linkedin/photon/ml/util/DaysRangeTest.scala

```diff
@@ -57,7 +58,9 @@ class DaysRangeTest {
   def testToDateRange(): Unit = {
 
     val expected = DateRange.fromDateString("20160102-20160331")
-    val actual = DaysRange.fromDaysString("90-1").toDateRange
+
+    // Today is 4/2 in UTC and 4/1 in America/Los_Angeles. One day ago should be 3/31 in America/Los_Angeles
+    val actual = DaysRange.fromDaysString("90-1").toDateRange(DateTimeZone.forID("America/Los_Angeles"))
```

Comment:
Lets add one more test which doesn't pass in the PST time zone, and checks that it will instead return `20160103-20160401`.

Reply:
Done.